views/playground/login/neutron.html.twig

```diff
@@ -32,7 +37,7 @@
 							type="text"
 							placeholder="username"
 							autocomplete="off"
-							tabindex="1"
+							tabindex="0"
```

Comment:
I could be wrong here but I'm fairly certain `tabindex="0"` is the same as not declaring a tabindex, so I think it safe to remove those attributes altogether in this file.

Reply:
It's an example UI that's already implemented in product, I wouldn't risk changing it in-product and breaking any js/tests for no benefit